be/btree.c

```diff
@@ -13159,6 +13170,261 @@ static void ut_btree_crc_persist_test_internal(struct m0_btree_type   *bt,
 	btree_ut_fini();
 }
 
+static void ut_mtree_mthread_test(void)
+{
+	int                          rc;
+	int                          ksize;
+	int                          vsize;
+	struct btree_ut_thread_info *ti;
+	int                          i;
+	int                          k;
+	int                          cur_arr_id = 0;
+	uint16_t                     cpu;
+	void                        *rnode;
+	struct m0_btree_op           b_op       = {};
+	struct btree_mthreads_data {
+		struct m0_btree_type   btree_type;
+		uint64_t               node_size;
+		uint64_t               no_of_trees;
+	} btrees_mthreads[] = {
+		{
+			{
+				M0_BT_BALLOC_GROUP_EXTENTS,
+				M0_MEMBER_SIZE(struct m0_ext, e_start),
+				M0_MEMBER_SIZE(struct m0_ext, e_end)
+			},
+			4096,
+			1
+		},
+		{
+			{
+				M0_BT_BALLOC_GROUP_DESC,
+				8,
+				96
```

Comment:
Please derive ksize and vsize from the structures, refer below snippet
`
.ksize = M0_MEMBER_SIZE(struct m0_balloc_group_desc,
						bgd_groupno),
.vsize = sizeof(struct m0_balloc_group_desc),
`

Reply:
Yes, tried that it does not work I get error trying to get size of a incomplete type.

Reply:
Please include "balloc/balloc.h" file.